modules/apps/depot/depot-web/src/main/resources/META-INF/resources/js/addConnectedSiteButtonPropsTransformer.js

```diff
@@ -0,0 +1,42 @@
+export default function propsTransformer({
+	currentURL,
+	eventName,
+	itemSelectorURL,
+	portletNamespace,
+	...otherProps
+}) {
+	return {
+		...otherProps,
+		onClick: () => {
+			Liferay.Util.selectEntity(
+				{
+					dialog: {
+						constrain: true,
+						destroyOnHide: true,
+						modal: true,
+					},
+					eventName,
+					id: `_${portletNamespace}_selectSite`,
+					title: Liferay.Language.get('select-site'),
+					uri: itemSelectorURL,
+				},
+				function (event) {
+					var toGroupIdInput = document.querySelector(
```

Comment:
This doesn't need to be `var` (same below).

Reply:
It's nitpicking time already, I see... that means the finish line is getting closer!! 😂 

![finishline](https://user-images.githubusercontent.com/905006/85018342-51ca8580-b16d-11ea-9f0b-ceb0127f9a57.gif)

Reply:
Ended up not changing `buttons` `onClick` yet in this PR... will make sure not to do this mistake again :D